submissions/Nik3264/js-dom/style.css

```diff
@@ -0,0 +1,255 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Oswald", sans-serif;
+}
+
+:root {
+  --light-brown: #8d7361;
+  --light-gray: #eeebe7;
+  --dark-brown: #50423d;
+  --grey-blue: #8b8da0;
+  --light-beige: #c9bbae;
+}
+
+@keyframes fade {
+  0% {
+    transform: translateX(100px);
+    opacity: 0;
+  }
+  100% {
+    transform: translateX(0);
+    opacity: 1;
+  }
+}
+
+@keyframes fade-text {
+  0% {
+    transform: translateY(-50px);
+    opacity: 0;
+    -webkit-clip-path: polygon(100% 0, 100% 100%, 0 100%, 0 100%);
```

Comment:
Remove` clip-path`.
Change `translateY(-50px);` to `translateY(-25px);`
Change `opacity: 0;` to `opacity: 0.75;`

Reply:
done